r/R/read_parquet.R

```diff
@@ -0,0 +1,10 @@
+#' Read parquet file from disk
+#' 
+#' @param files a vector of filenames
+#' @export
+read_parquet = function(files) {
+  tables = lapply(files, function(f) {
+    return (as_tibble(shared_ptr(`arrow::Table`, read_parquet_file(f))))
```

Comment:
Maybe we could have only *one* parquet reader instead of one per file. But I need to read more about the parquet api first.